input/input.css

```diff
@@ -0,0 +1,157 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800&display=swap");
+:root {
+	font-family: "Montserrat", sans-serif;
+	padding: 0px;
+	margin: 0px;
```

Comment:
My personal thought, I don't use units (px or rem for example) when the value is "0" for margin and padding :) 
It works!